index.html

```diff
@@ -84,5 +84,14 @@ <h2>Project</h2>
 	</div>
 </div>
 </section>
+
+	<footer>
+		<div class="contacts">
+			<h4>To Contact:-</h4>
```

Comment:
this pops out of the contacts bar when resizing browser (not sure why, sorry!)

Reply:
This is caused by the media query in styles.css lines 220-236.

``@media screen and (max-width: 900px) {
	
	h4 {
		margin: 5% 0 0 0;
	}
	
	.contacts {
		margin: 0 0 4% 0;
	}
}

@media screen and (max-width: 850px) {
	
	footer a {
		margin: 4% 0 0 0;
	}
}``